usr/share/rear/prep/GNU/Linux/305_include_s390_tools.sh

```diff
@@ -0,0 +1,28 @@
+#
+#  s390 zIPL boot loader and grubby for configuring boot loader`
+
+test -d $VAR_DIR/recovery || mkdir -p $VAR_DIR/recovery
+
+local bootdir="$( echo -n /boot/ )"
+test -d "$bootdir" || $bootdir='/boot/'
+
+# cf. https://github.com/rear/rear/issues/2137
+# findmnt is used the same as grub-probe to find the device where /boot is mounted
+# example
+# findmnt -no SOURCE --target /boot
+# --> /dev/dasda1
+#
+# sles will need to be looked at:
+# findmnt returns --> /dev/dasda3[/@/.snapshots/1/snapshot]
```

Comment:
Please check on SuSE whether this is an issue or not.

Reply:
grub2-probe works on suse, i think the way to go is ot use grub2-probe on suse, and thus will not see snapshots

Reply:
i have noted in this file sles action needed and will add it soon

Reply:
I will merge it as is regardless whether or not
booting on SLES with its complicated default btrfs structure works.
I assume things work on SLES with simple ext2/3/4 root filesystem.
We can add support for the complicated SLES default btrfs structure
later as needed.